contracts/BalanceTrackerFixedPriceBase.sol

```diff
@@ -268,27 +268,4 @@ abstract contract BalanceTrackerFixedPriceBase {
     function processPayment() external returns (uint256, uint256) {
         return _processPayment(msg.sender);
     }
-
-    /// @dev Withdraws funds for a specific requester account.
-    function withdraw() external {
```

Comment:
This was affecting only requester balances, so can just be safely deleted.

Reply:
Where was the request for this - please link for ease of review

Reply:
Actually it says "context deleted", so not sure now if it's still applicable?
https://docs.google.com/document/d/1K7pD_ipEaxdsyTWwnWXrIoPptsH7qXSPV7HmHHdmxPA/edit?disco=AAABa31QMuM